packages/core/src/core/fetch.ts

```diff
@@ -575,7 +575,7 @@ export const calcFetchChanges = async (
   accountElements: ReadonlyArray<Element>,
   mergedAccountElements: ReadonlyArray<Element>,
   stateElements: elementSource.ElementsSource,
-  workspaceElements: elementSource.ElementsSource,
+  workspaceElements: ReadOnlyElementsSource,
```

Comment:
This is the correct according to main and because of wrong conflict merge it needs to be fixed there